tests/landIce/Enthalpy/input_kleiner_B.yaml

```diff
@@ -124,7 +124,7 @@ ANONYMOUS:
             Field Origin: File
             Field Value: 0.0
           Field 4:
-            Number Of Layers: 41
+            Number Of Layers: 81
```

Comment:
why did you change this?

Reply:
Because the actual data (the ascii file) has 81 layers. I am not entirely sure how things worked before, but some changes I made ended up exposing this.